core/controllers/suggestion.py

```diff
@@ -374,3 +374,39 @@ def get(self):
 
         self.values.update({'suggestions': [s.to_dict() for s in suggestions]})
         self.render_json(self.values)
+
+
+class UpdateSuggestionHandler(base.BaseHandler):
```

Comment:
Reference: https://github.com/oppia/oppia/pull/12424/files#r612926165
/cc @DubeySandeep

I mean the same route could have been used for POST requests as well if it is kept within the same handler.
1. **Are we expecting UpdateSuggestionHandler to only handle translation suggestions? If yes, please change the handler class name and docstring to highlight this!** - `UpdateSuggestionHandler` will be genaralized to to use question suggestions as well. Currently only utilized to use translation suggestions. In that case, suggestions need to be filtered by the type and will need to have a parameter specifying the type of the suggestion that the request contains. I have added a TODO comment regarding this.
2. **In future once we allow updating all the suggestion are we expecting to use `SuggestionHandler`?** - No

Reply:
> UpdateSuggestionHandler will be generalized to use question suggestions as well.

But the API URL is more tied to translation, right? So how is this going to be used for general suggestions in the future?

Reply:
@chris7716 Let's make these backend APIs general. I'm not saying you need to add functionality for actually editing question suggestions in the frontend in this PR (that part can be separated out), but let's have the backend part be fully general rather than have a partial solution that requires a future TODO.